docs/jegyzet/mongodb/index.md

```diff
@@ -429,6 +440,18 @@ foreach (var g in collection.Aggregate()
         Console.WriteLine($"\tProduct: {p.Name}");
 }
 ```
+<br>
```

Comment:
HTML ne szerepeljen a markdown-ban. Nem csak HTML-be renderelhető a tartalom. Az újsor egyszerűen csak egy vagy két enter markdown-ban.